src/main/java/chatbot/tasks/Event.java

```diff
@@ -0,0 +1,72 @@
+package chatbot.tasks;
+
+import java.time.DateTimeException;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * Represents an event
+ */
+public class Event extends Task {
+    /**
+     * End date of event task
+     */
+    protected String end;
+    /**
+     * Start date of event task
+     */
+    protected String start;
+
+    /**
+     * Formatted start date of event task
+     */
+    protected LocalDateTime formattedStart;
+    /**
+     * Formatted end date of event task
+     */
+    protected LocalDateTime formattedEnd;
```

Comment:
I'm not entirely sure why these are currently `protected`, maybe they could be set to `private`?
```suggestion
    /**
     * End date of event task
     */
    private String end;
    /**
     * Start date of event task
     */
    private String start;

    /**
     * Formatted start date of event task
     */
    private LocalDateTime formattedStart;
    /**
     * Formatted end date of event task
     */
    private LocalDateTime formattedEnd;
```